src/qibo/tests_new/test_tensorflow_custom_operators.py

```diff
@@ -4,7 +4,7 @@
 import pytest
 import numpy as np
 from qibo import K, get_threads
-from qibo.tensorflow import custom_operators as op
```

Comment:
Another possibility to avoid importing this in every test is to just use `K.op`. `K.op` should provide access to the custom operators if `K` is the Tensorflow backend and raises an error if `K` is numpy, but this file is skipped for numpy anyway.

Reply:
Thanks, fixed.